src/js/pages/UniversePage.js

```diff
@@ -1,7 +1,21 @@
+import {MountService} from 'foundation-ui';
 import React from 'react';
 
 import Icon from '../components/Icon';
 
+const IntroductoryText = () => {
+  return (
+    <p className="lead tall">
+      Create a containerized service or run a command in one of two ways: use our form to be guided through the correct configuration, or enter your JSON configuration directly. You may also <a href="#/universe">browse the package repository</a>.
```

Comment:
Let's try to no replace the complete copy and only include the "link", as I could image that we eventually want to add more links.